program/src/processor.rs

```diff
@@ -519,52 +521,82 @@ pub fn process_update_exchange_rate(
     lido.save(accounts.lido)
 }
 
+pub enum StakeType {
+    Stake,
+    Unstake,
+}
+
+impl std::fmt::Display for StakeType {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            StakeType::Stake => write!(f, "stake"),
+            StakeType::Unstake => write!(f, "unstake"),
+        }
+    }
+}
+
+pub struct WithdrawExcessOpts<'a, 'b> {
+    accounts: &'a WithdrawInactiveStakeInfo<'a, 'b>,
+    clock: &'a Clock,
+    rent: &'a Rent,
+    stake_history: &'a StakeHistory,
+    stake_account: &'a AccountInfo<'b>,
```

Comment:
What is the difference between this account, and the other `accounts`?

Reply:
Ooh never mind, I was confused. The accounts are just the original accounts for the call.